tests/acceptance/features/apiSharingNg/deletePermissions.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Delete access to a drive item
+  https://owncloud.dev/libre-graph-api/#/drives.permissions/DeletePermission
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+    And using spaces DAV path
+
+
+  Scenario Outline: user removes access from a resource in the user share
```

Comment:
is it delete/revoke share?

Reply:
I'm using the exact wording from swagger https://owncloud.dev/libre-graph-api/#/drives.permissions/DeletePermission
it says `Remove access to a DriveItem.`  but in practicality yes it deletes the share
```suggestion
  Scenario Outline: user removes access to a resource in the user share
```

Reply:
Noice!